frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/contextmenu/SlideOutSelectedBranchAction.java

```diff
@@ -31,15 +31,19 @@
   public void update(AnActionEvent anActionEvent) {
     super.update(anActionEvent);
 
-    var presentation = anActionEvent.getPresentation();
```

Comment:
Any reason to not extract `presentation` to a var? Can `getPresentation()` return different objects each time? Although AFAICS `com.intellij.openapi.actionSystem.AnActionEvent#myPresentation` is final

Reply:
Same for `RebaseSelectedBranchOntoParentAction`

Reply:
As I had been changing this file anyway I wanted to provide consistency of the access to the presentation

Reply:
Or probably extracting presentation makes the code more readable... :thinking: